rss/nonerc-last-call.xml

```diff
@@ -11,7 +11,7 @@ layout: null
     <lastBuildDate>{{ site.time | date_to_rfc822 }}</lastBuildDate>
     {% assign pips = site.pages | sort: 'pip' %}
     {% for pip in pips %}
-      {% unless pip.category == "ERC" %}
+      {% unless pip.category == "PIP" %}
```

Comment:
Category can't be PIP, based on the above definition categories are: "Core, Networking, Interface"